transform-plugins/docs/Compressor-transform.md

```diff
@@ -3,26 +3,27 @@
 
 Description
 -----------
```

Comment:
please provide end user use case here as well . 
Like : apply any source -> compressor transform -> then any sink  (convert this into statements.)